src/hid/audio.cpp

```diff
@@ -32,6 +32,7 @@ class AudioHandle::Impl
     AudioHandle::Result Init(const AudioHandle::Config config, SaiHandle sai);
     AudioHandle::Result
                         Init(const AudioHandle::Config config, SaiHandle sai1, SaiHandle sai2);
+    AudioHandle::Result Deinit();
```

Comment:
I don't want to mark every single one, but I think `DeInit` is a bit more common for naming convention than `Deinit`